lib/ansible/plugins/inventory/scaleway.py

```diff
@@ -27,6 +27,12 @@
         tags:
             description: Filter results on a specific tag
             type: list
+        mandatory_tags:
+            description: Filter result always matching all these tags - Added in Ansible 2.8
```

Comment:
Use [`version_added`](https://docs.ansible.com/ansible/latest/dev_guide/developing_plugins.html#plugin-configuration-documentation-standards) instead of mentioning the version in the description.